timelock-agent/src/test/java/com/palantir/timelock/paxos/TimeLockAgentTest.java

```diff
@@ -229,10 +237,11 @@ private File getMockFileWith(boolean isDirectory, boolean canCreateDirectory) th
 
     private void assertFailsToVerifyConfiguration(ImmutablePaxosInstallConfiguration.Builder configBuilder) {
         PaxosInstallConfiguration installConfiguration = configBuilder.build();
-        assertThatThrownBy(() -> TimeLockAgent.verifyIsNewServiceInvariant(TimeLockInstallConfiguration.builder()
-                        .cluster(CLUSTER_CONFIG)
-                        .paxos(installConfiguration)
-                        .build()))
+        assertThatThrownBy(() -> TimeLockAgent.verifyIsNewServiceInvariant(
+                        TimeLockInstallConfiguration.builder()
+                                .paxos(installConfiguration)
+                                .build(),
+                        CLUSTER_CONFIG))
                 .isInstanceOf(IllegalArgumentException.class);
     }
 }
```

Comment:
I might have missed this, but somewhere here we probably want a test that verifies that the old mechanism works (i.e. a test where install configuration says to allow a nonstandard topology but the cluster config is 1 or 2 nodes and does not - this should not fail).